app/src/main/java/com/framgia/forder/data/model/ShopManagement.java

```diff
@@ -0,0 +1,84 @@
+package com.framgia.forder.data.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports